h/static/styles/annotation.scss

```diff
@@ -257,6 +175,26 @@ $annotation-card-left-padding: 10px;
   }
 }
 
+.share-dialog-wrapper {
+  position: relative;
+
+  &.open .share-dialog--actions {
+    display: block;
+  }
+}
+
+.share-dialog--actions {
+  padding: 3px;
+  width: inherit;
+}
+
+.share-dialog__link {
+  padding: 0;
+  width: 220px;
```

Comment:
I generally prefer to use `min-width` and `min-height` to set minimum dimensions for a component with a bunch of sub-elements. This avoids the risk of cutting off content if the component's natural size is larger than the minimum for any reason - eg. a label being longer than anticipated.